Buildings/Resources/Documentation/userGuide/source/development.rst

```diff
@@ -170,18 +539,38 @@ The following rules need to be followed, in addition to the guidelines described
    Do not use ``Dialog(tab=STRING))``, unless the parameter is declared with a default value
    and this parameter and its value is of no interest to typical users.
 
-#. Each block must have a ``defaultComponentName`` annotation.
+#. In the code text the instances must be ordered as follows:
+     - If an instance has a type associated with it, then list them in this order: `boolean`, `integer`, `real`.
+     - Instances of lower dimensionality should come first, e.g. scalar values before arrays, 
+       though grouping based on model specific similarities may be prioritized
+     - Instances should be ordered as: parameters, interfaces: first inputs, then outputs, then ports,
```

Comment:
There are no `ports` in CDL  and simply call them inputs and outputs, not interfaces as interface is not defined what it is. Also, correct punctuation to use a full sentence.

Reply:
`ports` is legacy from previous version of the document, as it is for both CDL and other development within the library.